tests/unit_tests/test_client.py

```diff
@@ -1078,3 +1079,37 @@ def test_identification_headers_unique_connection_id():
         "UNLEASH-CONNECTION-ID"
     ]
     assert connection_id_first_client != connection_id_second_client
+
+
+@responses.activate
+def test_identification_values_are_passed_in():
```

Comment:
This is testing all 3 requests, headers and body

Reply:
I remember we had this test added: https://github.com/Unleash/unleash-client-python/pull/342/files#diff-6fd6de24505791fbd1ad578927a8fdd87d15a14b7d194600d68434ddc3e53e00R1038 Are those 2 tests testing similar things?